src/lib/util.ts

```diff
@@ -0,0 +1,25 @@
+interface AnyObject {
+  [key: string]: any;
+}
+
+/**
+ * Sorts an object
+ * @param obj
+ */
+export function sortObject(obj: object): object {
```

Comment:
What is the semantics of this function meant to be?

Reply:
What do you mean? Does changing the description to `Returns a new clone of 'obj' with its keys sorted alphabetically` answer that question?